navbar.css

```diff
@@ -2,39 +2,22 @@ nav {
 	display: flex;
 	flex-direction: row-reverse;
 	justify-content: space-between;
-}
-
-ul {
-	
-	color: black;
+	color: #ffe4e1;
 }
 
 li {
 	padding: 0.5em;
 	display: inline;
-	color: #ffffff;
-	
+	color: #ffe4e1;
 }
 
+li a {
+	color: #ffe4e1;
+}
 a:visited {
 	color: #ffe4e1;
```

Comment:
I don't think you need this when you've already set the links to be in this color within the li a selector